packages/frontend/src/main/components/common/StreamPreviewCard.vue

```diff
@@ -63,14 +67,25 @@
 <script>
 export default {
   components: {
-    PreviewImage: () => import('@/main/components/common/PreviewImage'),
-    CollaboratorsDisplay: () => import('@/main/components/stream/CollaboratorsDisplay')
+    PreviewImage: () => import('@/main/components/common/PreviewImage.vue'),
```

Comment:
VSCode expects me to add the .vue extension for it to be able to understand what file this points to